lib/internal/errors.js

```diff
@@ -792,6 +792,10 @@ E('ERR_FALSY_VALUE_REJECTION', function(reason) {
   this.reason = reason;
   return 'Promise was rejected with falsy value';
 }, Error);
+E('ERR_FEATURE_UNAVAILABLE_ON_PLATFORM',
+  'The feature %s is unavailable on the current platform' +
+  ', which is being used to run Node.js',
```

Comment:
Nit: the message could actually include the used platform. That way it would be easier to identify the used platform in case the logs would e.g., be transferred to a third party application that monitors the errors.

Reply:
I'm sorry, I currently don't have time for continue working on this PR. If someone else would like to make this change I'll appreciate it. thanks.